static/js/server_events_dispatch.js

```diff
@@ -313,17 +313,34 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             // than requiring a reload or page resize.
         }
         if (event.setting_name === 'emojiset') {
-            var sprite = new Image();
-            sprite.onload = function () {
-                $("#emoji-spritesheet").attr('href', "/static/generated/emoji/" + page_params.emojiset + "_sprite.css");
+            if (page_params.emojiset === 'text') {
+                page_params.emoji_alt_code = true;
```

Comment:
You should remove any occurrence of `emoji_alt_code` from frontend code replacing it by `emojiset`.(with some logic if required)

Reply:
emoji_alt_code is used in many test files and logic which will need to be changed. So I will need to make changes in a lot of files both in backend and in frontend code including tests. Should I make all those changes in all the files?

Reply:
Yep!